src/Jackett.Common/Definitions/vizuk.yml

```diff
@@ -6,8 +6,6 @@
   type: private
   encoding: UTF-8
   links:
-    - http://torrent.vizuk.li/
```

Comment:
don't remove the legacylinks, just flip the http / https.
this will automatically update users vizuk's indexer profile so they don't have to manually change the site link themselves.